tools/build-tool/data/android/integrated/app/build.gradle

```diff
@@ -10,13 +10,21 @@ android {
         versionCode ${VERSION_CODE}
         versionName "${VERSION}"
 
+        ndkVersion "22.1.7171670"
```

Comment:
Do we need to specify this exact version? Seems very specific.

Reply:
By default gradle used 21 but I just wanted use the same I precompiled libraries before (to make easier testing). So we don't need it. But we should add ability to choose it in our manifest file (as far as I know we don't support that now). That should help someone with close sourced c++ libraries that was compiled with specified ndk version to use the same version with our libraries.

Reply:
https://developer.android.com/ndk/guides/cpp-support#one_stl_per_app